website/src/_data/capability-matrix.yml

```diff
@@ -359,10 +385,10 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3:
-          - class: spark
```

Comment:
Why it was deleted and not renamed to `spark-rdd` as you did above?

Reply:
It is an error, good catch ! thanks